rln/benches/pmtree_benchmark.rs

```diff
@@ -40,5 +50,21 @@ pub fn pmtree_benchmark(c: &mut Criterion) {
     });
 }
 
-criterion_group!(benches, pmtree_benchmark);
+pub fn key_load_benchmark(c: &mut Criterion) {
+    #[cfg(feature = "arkzkey")]
+    c.bench_function("ark_zkey::load", |b| {
+        b.iter(|| {
+            let _ = arkzkey_from_folder(TEST_RESOURCES_FOLDER);
+        })
+    });
+
+    #[cfg(not(feature = "arkzkey"))]
+    c.bench_function("zkey::load", |b| {
+        b.iter(|| {
+            let _ = rln::circuit::zkey_from_folder(TEST_RESOURCES_FOLDER);
```

Comment:
maybe write in all possible calls like this? I mean if we have only one call in file we can avoid long definition of dependencies like in line 8-14 above

@rymnc  what do you think?

Reply:
hmm agree, also, sorry for the nit, maybe we can move this benchmark into a new file in the benches directory "circuit_loading_benchmarks.rs", since this is not directly related to the pmtree benchmarks

Reply:
sure, np